linera-service/src/cli/main.rs

```diff
@@ -1222,33 +1335,17 @@ impl Runnable for Job {
     about = "A Byzantine-fault tolerant sidechain with low-latency finality and high throughput",
 )]
 struct ClientOptions {
+    /// Common options.
+    #[command(flatten)]
+    context: ClientContextOptions,
```

Comment:
Should we name it `context_options`? (Or call the other field `common_storage`?)

Reply:
fair